sass_processor/management/commands/compilescss.py

```diff
@@ -5,7 +5,7 @@
 from optparse import make_option
 from django.conf import settings
 from django.core.management.base import BaseCommand, CommandError
-from django.template.loader import get_template  # noqa Leave this in to preload template locations
+from django.template.loader import get_template, Origin  # noqa Leave get_template in to preload template locations
```

Comment:
Just had a look into Django-1.8, here ``Origin`` is part of ``django.template.base``